content/departments/engineering/managed-services/build-tracker.md

```diff
@@ -34,6 +43,7 @@ If you need assistance with MSP infrastructure, reach out to the [Core Services]
 | ------------------- | ------------------------------------------------------------------------------------------------------ |
 | Project ID          | [`build-tracker-prod-59bf`](https://console.cloud.google.com/run?project=build-tracker-prod-59bf)      |
 | Category            | **test**                                                                                               |
+| Deployment Type     | `rollout`                                                                                              |
```

Comment:
Hmm, should use sentence case to be consistent?

```suggestion
| Deployment type     | `rollout`                                                                                              |
```

Reply:
Yes, I always forget 🤦‍♂️